core/src/test/java/org/apache/iceberg/TestUpdateRequirements.java

```diff
@@ -198,11 +201,26 @@ public void upgradeFormatVersion() {
     assertTableUUID(requirements);
   }
 
+  @ParameterizedTest
+  @ValueSource(ints = {2, 3})
+  public void upgradeFormatVersionForView(int formatVersion) {
```

Comment:
actually we don't need to adjust any of the tests for views and this test can be removed, because views are currently at format-version = 2

Reply:
Makes sense, reverting the test